Exercises-2-Basic/src/exercises/Num1.java

```diff
@@ -0,0 +1,58 @@
+/* 1.
+Write a Java program to print the sum (addition), multiply, subtract, divide and 
+remainder of two numbers.
+Test Data:
+Input firstnumber: 125
+Input secondnumber: 24
+Expected Output :
+125 + 24 = 149
+125 - 24 = 101
+125 x 24 = 3000
```

Comment:
This description should be above the class declaration, meaning just above the line:
`public class Num1 {`

Reply:
Done

Reply:
Check other classes as well please.

Reply:
okay done

Reply:
not done!

Reply:
done.